### DISCUSSION TITLE
don't allow creation of duplicate prefixes

### DISCUSSION THREAD

**Discussion by dbljakaj** [netbox-community/netbox#15195] [Q&A]
Hello,

I saw that I can create two times the same subnet within the global VRF.
Is it possible to block the creation for users if the subnet already exists and how ?

Version : v3.6.4

Thank in advance.

**Comment by dbljakaj** [comment]
Found it myself, I've to create a specific VRF for it

**Reply by jeremystretch** [reply]
Or you can set [`ENFORCE_GLOBAL_UNIQUE`](https://docs.netbox.dev/en/stable/configuration/miscellaneous/#enforce_global_unique) to true.

**Reply by dbljakaj** [reply]
I think I broke the system while adding this line to configuration.py
I then restarted netbox and netbox-rq, 

For netbox, I'm getting : 
      systemd[4884]: netbox.service: Failed at step EXEC spawning /opt/netbox/venv/bin/gunicorn: No such file or directory

For netbox-rg , I'm getting : 
        python3[4778]: Traceback (most recent call last):
        python3[4778]:   File "/opt/netbox/netbox/manage.py", line 8, in <module>
        python3[4778]:     from django.core.management import execute_from_command_line
        python3[4778]: ModuleNotFoundError: No module named 'django'
        systemd[1]: netbox-rq.service: Main process exited, code=exited, status=1/FAILURE


I also rebooted but the services are still not up

**Reply by dbljakaj** [reply]
Thanks for your help @jeremystretch 
Any idea for my problem that came after?
